internal/pkg/cli/svc_init.go

```diff
@@ -712,6 +747,7 @@ This command is also run as part of "copilot init".`,
 	cmd.Flags().Uint16Var(&vars.port, svcPortFlag, 0, svcPortFlagDescription)
 	cmd.Flags().StringArrayVar(&vars.subscriptions, subscribeTopicsFlag, []string{}, subscribeTopicsFlagDescription)
 	cmd.Flags().BoolVar(&vars.noSubscribe, noSubscriptionFlag, false, noSubscriptionFlagDescription)
+	cmd.Flags().StringVar(&vars.ingressType, ingressTypeFlag, "", ingressTypeFlagDescription)
```

Comment:
What do you think of setting the default to "Environment"? There are a bunch of script users and this change will break their script.

Reply:
Normally I'd say the most private option should be the default, but should "Internet" be the default in this case because it's currently the only option?

Reply:
if we have default, are you thinking to remove the prompt..?

Reply:
discussed offline, we'll leave the prompt for now. script users will have to add the flag.